src/App.jsx

```diff
@@ -1,34 +1,49 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+
 import './App.scss';
 import './styles/general.scss';
+
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api/api';
 
-class App extends React.Component {
-  state = {
-    todos: [],
-    selectedUserId: 0,
-  };
+const App = () => {
+  const [todos, setTodos] = useState([]);
+  const [selectedUserId, setUserId] = useState(0);
 
-  render() {
-    const { todos, selectedUserId } = this.state;
+  useEffect(() => {
+    getTodos(todos)
+      .then(setTodos);
+  }, []);
 
-    return (
-      <div className="App">
-        <div className="App__sidebar">
-          <TodoList todos={todos} />
-        </div>
+  const clearUser = () => {
+    setUserId(0);
+  };
+
+  return (
+    <div className="App">
+      <div className="App__sidebar">
+        <TodoList
+          todos={todos}
+          selectedUserId={selectedUserId}
+          selectUser={(userId) => {
+            setUserId(userId);
```

Comment:
For better reading you may write a function `selectUser` and just call it here:
`selectUser={selectUser}`